draft-ietf-rtcweb-jsep.xml

```diff
@@ -1178,7 +1178,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           on when it is invoked, and differ depending on which specific
           parameters are changed:</t>
          <t><list style="symbols">
-            <t>Any changes to the STUN/TURN servers to use affect
+            <t>Any changes to the STUN/TURN servers to use will affect
```

Comment:
hmm - seems like we were missing a verb as I was thinking affect is noun but will defer to someone who speaks english